test/PspHgh.jl

```diff
@@ -82,7 +81,13 @@ end
 
 end
 
-@testset "Check qcut routines" begin
+@testitem "Check qcut routines" tags=[:psp] begin
+    using LinearAlgebra
+    using DFTK: load_psp, eval_psp_projector_fourier, eval_psp_local_fourier
+    using DFTK: eval_psp_projector_real, psp_local_polynomial, eval_psp_local_real
+    using DFTK: psp_projector_polynomial, qcut_psp_projector, qcut_psp_local
+    using DFTK: eval_psp_energy_correction, count_n_proj_radial
```

Comment:
Would be nice to be more specific here and not just duplicate the same blob a couple of times ...